translations/es.ts

```diff
@@ -2142,7 +2142,7 @@ Será instalada cuando reinicies qTox.</translation>
     <message>
         <location filename="../src/widget/form/removefrienddialog.ui" line="39"/>
         <source>YES</source>
-        <translation>SI</translation>
+        <translation>SÍ</translation>
```

Comment:
captcha? tecleo?
I'm a translator, not a coder. So, you are saying, the YES needs to be typed in by the user?